src/main/resources/db/migrationv2/V2_13__summoning_updates.sql

```diff
@@ -0,0 +1,30 @@
+drop function juror_mod.get_voters(bigint,text, text, text, text, text);
+
+alter table juror_mod.voters
+    drop column postcode_start,
+    add column postcode_start VARCHAR(10) generated always as (split_part(zip, ' ', 1)) stored;
+
+DROP INDEX juror_mod.voters_postcode_start_idx;
```

Comment:
dont need this line for a fresh deployment

Reply:
Removed :)